packages/components/Menu/src/MenuGroup/MenuGroup.tsx

```diff
@@ -1,18 +1,25 @@
 /** @jsxRuntime classic */
 /** @jsx withSlots */
 import React from 'react';
-import { View } from 'react-native';
+import { Platform, View } from 'react-native';
 
+import { FocusZone } from '@fluentui-react-native/focus-zone';
 import { compose, mergeProps, withSlots } from '@fluentui-react-native/framework';
 import type { UseSlots } from '@fluentui-react-native/framework';
 
 import type { MenuGroupProps, MenuGroupType } from './MenuGroup.types';
 import { menuGroupName } from './MenuGroup.types';
 
+// macOS has focus zone around entire list, including it for groups may
+// cause bad keyboard behavior. Will review in the future.
+const hasFocusZone = ['win32'].includes(Platform.OS as string);
```

Comment:
Nice, silence typescript check lol

Reply:
Ah I think I had this as "[macos, win32].includes" before and then I removed the "macOS". but this'll make it easy to add back if it ever needs to be